app/mailers/patient_mailer.rb

```diff
@@ -69,7 +71,29 @@ def assessment_sms_weblink(patient)
       params = { prompt: contents, patient_submission_token: patient.submission_token,
                  threshold_hash: threshold_hash, medium: 'SINGLE_SMS' }
       patient.update(last_assessment_reminder_sent: DateTime.now) # Update last send attempt timestamp before Twilio call
-      add_success_history(dependent) if TwilioSender.send_sms(patient, params)
```

Comment:
This looks like old code. As of now there is an `else` case that this PR would also need to address here: https://github.com/SaraAlert/SaraAlert/blob/master/app/mailers/patient_mailer.rb#L73

Reply:
The twilio_error_handling branch doesn't operate the same way that the assessment_sms_weblink function does in the  link you sent. -- This is another argument in support of just having one larger twilio_error_handling PR